components/database/mariadb-101/patches/01-cfi-issue.patch

```diff
@@ -0,0 +1,22 @@
+--- mariadb-10.1.17/mysys/my_context.c.~1~	2016-07-27 19:23:07.480630772 +0300
```

Comment:
I don't see something like https://github.com/OpenIndiana/oi-userland/blob/oi/hipster/components/database/mariadb-55/patches/03-my.cnf-serch-path.patch , so that server doesn't try reading /etc/mysql/my.cnf, when /etc/mariadb/10.1/my.cnf is present.